packages/solana/lib/src/rpc/dto/account_data/saved.g.dart

```diff
@@ -0,0 +1,24 @@
+// GENERATED CODE - DO NOT MODIFY BY HAND
```

Comment:
Is this file needed at all?

Reply:
Not at all. It was a temporary file. How did you see it? :fire: